mandatory/3-playing-computer.js

```diff
@@ -7,28 +7,39 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+     there is no value and use for b .
   2. Remove the line that throws the error.
   3. What is printed to the console?
+     2
+     6
+     2
+     9
+     4
+     13
+     6
   4. How many times is "f1" called?
+     once
   5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
+     once
+  6. What value does the "a" parameter take in the first "f1" call? 
+     6
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
-*/
+*/  7
 
 let x = 2;
 let a = 6;
 
-const f1 = function (a, b) {
-  return a + b;
+const f1 = function (a, x) {
+  return a + x;
 };
 
-const f2 = function (a, b) {
-  return a + b + x;
+const f2 = function (a, x) {
+  return a + x;
 };
 
 console.log(x);
 console.log(a);
-console.log(b);
+
```

Comment:
I don't think you are supposed to make changes to this code. `b` was supposed to throw an error as you suggested